plugin/EasyGrep.vim

```diff
@@ -1973,6 +1973,11 @@ function! s:ParseCommandLine(argv)
     let opts["failedparse"] = ""
     let parseopts = 1
 
+    if exists("g:EasyGrepDisableCmdParam") && g:EasyGrepDisableCmdParam==1
```

Comment:
sorry, hasn't noticed that, however, i tried this command:

```
:Grep -- - 1
```

and got this error:

```
grep: unknown option -- ' '
```

grepCommand (around line 2697) seems nothing wrong